psi4/src/psi4/lib3index/df_helper.cc

```diff
@@ -137,42 +137,51 @@ void DF_Helper::initialize()
 {
     outfile->Printf("\n\n    ==> DF_Helper: initialize() <==\n\n");
     timer_on("DF_Helper~-- build-------   ");
-    if(method_.compare("DIRECT") && method_.compare("STORE") && method_.compare("AO_CORE")){
+    if(method_.compare("DIRECT") && method_.compare("STORE")){
         std::stringstream error;
         error << "DF_Helper:initialize: specified method (" << method_ << ") is incorrect";
         throw PSIEXCEPTION(error.str().c_str());
     }
+    direct_ =  (!method_.compare("DIRECT" ) ? true : false);
 
     outfile->Printf("      nao           = %zu\n", nao_);
     outfile->Printf("      naux          = %zu\n", naux_);
     outfile->Printf("      Scwarz cutoff = %E\n", cutoff_);
     outfile->Printf("      mem (doubles) = %zu\n", memory_);
-    outfile->Printf("      method        = %s\n", method_.c_str());
-    outfile->Printf("      core?         = %d\n", (int)core_);
+    outfile->Printf("      nthreads      = %zu\n", nthreads_);
+    outfile->Printf("      Algorithm     = %s\n", method_.c_str());
+    outfile->Printf("      AO_core?      = %d\n", (int)AO_core_);
+    outfile->Printf("      MO_core?      = %d\n", (int)MO_core_);
```

Comment:
We might want to cleanup the text a bit and make sure it looks very similar to the current DF-JK and DF Tensor print outs. One thing to note is the `=> Something <=` headers.